### PR TITLE
Update help wiki

### LINKED ISSUE #33: Offer built-in tutorial
As a <kbd>*new user*</kbd> I would like a tutorial built into the image in order to help me quickly understand the tool upon first use. Currently the test-runner is quite cluttered, especially for a new user, which could be remedied through this. 

### Acceptance criteria

- [ ] A new user should be able to execute, edit, and add tests comfortably after working through the tutorial for the **first** time  
- [ ] The tutorial explains how to add and run custom tests
- [x] The tutorial is offered as an initial prompt upon opening the test-runner for the first time. Afterwards this prompt should not be displayed

### Effort Evaluation

Points: 4

### PR DESCRIPTION
**Pull request by Toalaah**
Closes: #33

Fixes minor spelling / grammar mistakes in the ATDD help wiki. Adds missing GitHub actions entry.

### CODE REVIEW

File: packages/AutoTDD-Help.package/ATDDHelp.class/class/browser.st [hunk 1]
```diff
@@ -4,6 +4,6 @@ browser
 	^ HelpTopic
 		title: 'Browser'
 		contents: 
-			'The Browser is our main GUI which will show all the Watchers and their current state as well as all Travis Statuses with the corresponding build state, the commit message, the branch and the author of the latest Travis build. It can also be used to create, edit and remove Watchers and Travis Statuses.
-		
-You can open the Browser under "Apps -> Test Auto Runner."'
\ No newline at end of file
+			'The browser is the primary form of interacting with AutoTDD. It contains a list of all of your locally defined watchers as well as any Travis statuses and GitHub actions which you have added. Testing entities are managed from this GUI; this includes adding, editing, removing, and inspecting testing entities. All of your watchers are saved upon closing the browser as well as your image.
```
Comment (benjamin-frost) [inline packages/AutoTDD-Help.package/ATDDHelp.class/class/browser.st]: nit

```suggestion
			'The browser is the primary form of interacting with AutoTDD. It contains a list of all of your locally defined watchers as well as any Travis statuses and GitHub Actions which you have added. Testing entities are managed from this GUI; this includes adding, editing, removing, and inspecting testing entities. All of your watchers are saved upon closing the browser as well as your image.
```

**Review by kolioOtSofia — APPROVED** [review]

File: packages/AutoTDD-Help.package/ATDDHelp.class/class/githubActions.st [hunk 2]
```diff
@@ -0,0 +1,10 @@
+content
+githubActions
+	
+	^ HelpTopic
+		title: 'Using GitHub Actions'
+		contents: 
+			'GitHub actions will show you the latest state, commit message, branch, and author of the most recent GitHub action run on your project. You can add a new GitHub action by clicking the ''Add GitHub Action'' button in the browser. You can provide an optional personal access token which will allow you to bypass the 60 request per hour limit. GitHub actions are  shown in the browser under the ''GitHub Actions'' section and will refresh their contents automatically within a user-defined interval.
```
Comment (benjamin-frost) [inline packages/AutoTDD-Help.package/ATDDHelp.class/class/githubActions.st]: nit

```suggestion
			'GitHub Actions will show you the latest state, commit message, branch, and author of the most recent GitHub Action run on your project. You can add a new GitHub Action by clicking the ''Add GitHub Action'' button in the browser. You can provide an optional personal access token which will allow you to bypass the 60 request per hour limit. GitHub Actions are  shown in the browser under the ''GitHub Actions'' section and will refresh their contents automatically within a user-defined interval.
```

File: packages/AutoTDD-Help.package/ATDDHelp.class/class/githubActions.st [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+			
+
+Clicking on the ''Results'' button open a new window showing all GitHub action runs and their corresponding logs.'
```
Comment (benjamin-frost) [inline packages/AutoTDD-Help.package/ATDDHelp.class/class/githubActions.st]: nit

```suggestion
Clicking on the ''Results'' button open a new window showing all GitHub Action runs and their corresponding logs.'
```

File: packages/AutoTDD-Help.package/ATDDHelp.class/class/overviewWindow.st [hunk 4]
```diff
@@ -0,0 +1,9 @@
+content
+overviewWindow
+	
+	^ HelpTopic
+		title: 'Overview Window'
+		contents: 
+			'The overview window is a minimal, unobtrusive UI which contains  a summary of the state of a user-defined subset of all local watchers, Travis statuses, and GitHub actions. In order to create an overview window you can open the browser, select at least one watcher, Travis status, or GitHub action, and then click on the ''Create Overview'' button on the top of the window.
```
Comment (benjamin-frost) [inline packages/AutoTDD-Help.package/ATDDHelp.class/class/overviewWindow.st]: nit

```suggestion
			'The overview window is a minimal, unobtrusive UI which contains a summary of the state of a user-defined subset of all local watchers, Travis statuses, and GitHub Actions. In order to create an overview window you can open the browser, select at least one watcher, Travis status, or GitHub Action, and then click on the ''Create Overview'' button on the top of the window.
```

File: packages/AutoTDD-Help.package/ATDDHelp.class/class/travisStatuses.st [hunk 5]
```diff
@@ -4,7 +4,7 @@ travisStatuses
 	^ HelpTopic
 		title: 'Using Travis Statuses'
 		contents: 
-			'You can create a Travis Status that will show you the build state, the commit message, the branch and the author of the latest Travis build of your repository. Travis Statuses are also
-shown in the Browser and will refresh their contents automatically within an interval that you can also specify.
+			'Travis statuses will show you the build state, commit message, branch, and author of the latest Travis build of your repository. You can add a new Travis status by clicking the ''Add Travis Status'' button in the browser. This is also where you can specifiy whether or not you want to use Travis pro, which requires a corresponding access token. Travis statuses are also shown in the browser under the ''Travis Statuses'' section and will refresh their contents automatically within a user-defined interval.
+
```
Comment (benjamin-frost) [inline packages/AutoTDD-Help.package/ATDDHelp.class/class/travisStatuses.st:8]:
```suggestion
```

**Review by benjamin-frost — CHANGES_REQUESTED** [review]

File: packages/AutoTDD-Help.package/ATDDHelp.class/class/githubActions.st [hunk 6]
```diff
@@ -0,0 +1,10 @@
+content
+githubActions
+	
+	^ HelpTopic
+		title: 'Using GitHub Actions'
+		contents: 
+			'GitHub Actions will show you the latest state, commit message, branch, and author of the most recent GitHub Action run on your project. You can add a new GitHub Action by clicking the ''Add GitHub Action'' button in the browser. You can provide an optional personal access token which will allow you to bypass the 60 request per hour limit. GitHub Actions are  shown in the browser under the ''GitHub Actions'' section and will refresh their contents automatically within a user-defined interval.
```
Comment (SebastianWagner2) [inline packages/AutoTDD-Help.package/ATDDHelp.class/class/githubActions.st]:
```suggestion
			'GitHub Actions will show you the latest state, commit message, branch, and author of the most recent GitHub Action run on your project. You can add a new GitHub Action by clicking the ''Add GitHub Action'' button in the browser. You can provide an optional personal access token which will allow you to bypass the 60 request per hour limit. GitHub Actions are  shown in the browser in the ''GitHub Actions'' section and will refresh their contents automatically within a user-defined interval.
```

File: packages/AutoTDD-Help.package/ATDDHelp.class/class/travisStatuses.st [hunk 7]
```diff
@@ -4,7 +4,6 @@ travisStatuses
 	^ HelpTopic
 		title: 'Using Travis Statuses'
 		contents: 
-			'You can create a Travis Status that will show you the build state, the commit message, the branch and the author of the latest Travis build of your repository. Travis Statuses are also
-shown in the Browser and will refresh their contents automatically within an interval that you can also specify.
-
-For Travis Statuses, a click on the Results button will show all Travis builds and the corresponding logs in a special GUI.'
\ No newline at end of file
+			'Travis statuses will show you the build state, commit message, branch, and author of the latest Travis build of your repository. You can add a new Travis status by clicking the ''Add Travis Status'' button in the browser. This is also where you can specifiy whether or not you want to use Travis pro, which requires a corresponding access token. Travis statuses are also shown in the browser under the ''Travis Statuses'' section and will refresh their contents automatically within a user-defined interval.
```
Comment (SebastianWagner2) [inline packages/AutoTDD-Help.package/ATDDHelp.class/class/travisStatuses.st]:
```suggestion
			'Travis statuses will show you the build state, commit message, branch, and author of the latest Travis build of your repository. You can add a new Travis status by clicking the ''Add Travis Status'' button in the browser. This is also where you can specifiy whether or not you want to use Travis pro, which requires a corresponding access token. Travis statuses are also shown in the browser in the ''Travis Statuses'' section and will refresh their contents automatically within a user-defined interval.
```

**Review by SebastianWagner2 — APPROVED** [review]

**Review by benjamin-frost — APPROVED** [review]

**Review by benjamin-frost — APPROVED** [review]

**Pull request merged**